src/coordinator/index.spec.js

```diff
@@ -51,6 +51,111 @@ describe('Coordinator', function() {
     });
   });
 
+  describe('createFavoriteApplication', function() {
+    context('when the application ID is provided', function() {
+      let expectedApplicationFavorite;
+      let applicationFavoriteFromServer;
+      let promise;
+      let request;
+      let toCamelCase;
+
+      beforeEach(function() {
+        expectedApplicationFavorite = fixture.build(
+          'contxtUserFavoriteApplication'
+        );
+        applicationFavoriteFromServer = fixture.build(
+          'contxtUserFavoriteApplication',
+          expectedApplicationFavorite,
+          {
+            fromServer: true
+          }
+        );
+
+        request = {
+          ...baseRequest,
+          post: this.sandbox.stub().resolves(applicationFavoriteFromServer)
+        };
+        toCamelCase = this.sandbox
+          .stub(objectUtils, 'toCamelCase')
+          .callsFake((app) => expectedApplicationFavorite);
```

Comment:
This works just fine so it doesn't have to change, but could use `returns` here instead of `callsFake`

Reply:
Fixed 👍